### PR TITLE
Updates a few dependencies to the latest compatible versions

### PR DESCRIPTION
**Pull request by adambharris**
The latest version of `cosmos:browserify` no longer seems to have the issues that caused its version number to be pegged to v0.5.0.

The forked version of `page.js` can be updated to v1.6.4, which includes support for path-to-regexp v1.2.1. That in particular solves a compatibility issue I faced when migrating an app with a single-asterisk catch-all route from Iron-Router. I'll need to update this pull request with a link to your version once my [pull request there](https://github.com/kadirahq/page.js/pull/3) is accepted.

The `qs` npm dependency can be updated to v5.2.0, but not yet to the latest v6.0.0.

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ Package.describe({
 Npm.depends({
   // In order to support IE9, we had to fork pagejs and apply
   // this PR: https://github.com/visionmedia/page.js/pull/288
-  'page':'https://github.com/kadirahq/page.js/archive/f29d4d4491178b285b9058c32d74975a4f945dea.tar.gz',
-  'qs':'3.1.0'
+  'page':'https://github.com/adambharris/page.js/archive/3f404dd90ee6d806af046e41b2faa195d940e629.tar.gz',
```
Comment (arunoda) [inline package.js]: Could you send me a PR to our project at `kadirahq/page.js` with the changes. I'd like to merge it.
Then we can use it.

**Comment by arunoda** [comment]
Could you send me a separate PR with all the grammar fixes. Then I can get them quickly.

File: package.js (hunk 1 above)
Comment (adambharris) [inline package.js]: Here you go: https://github.com/kadirahq/page.js/pull/3

**Comment by arunoda** [comment]
Okay. Now I merged the PR you sent to our page.js repo: https://github.com/kadirahq/page.js
Update this to use that one and I'll merge it then.

**Comment by arunoda** [comment]
Drop me a comment once you fix this. Github does not notify me when you make a change.

**Comment by adambharris** [comment]
Updated and ready to go.

**Comment by arunoda** [comment]
Hey @adambharris Did you merge this PR to master. Please don't do this.
I'll keep that to myself. :)

There are some manual checks I need to do before merging any PR.

**Pull request merged**